src/docs/config/overview.md

```diff
@@ -97,6 +98,13 @@ globalStyle: 'src/global/app.css'
 
 Check out the [styling docs](https://stenciljs.com/docs/styling#global-styles) of how to use global styles in your app.
 
+## invisiblePrehydration
+
```

Comment:
[Other fields have the default between the name of the field and the description](https://github.com/ionic-team/stencil-site/pull/761/files#diff-689c19470ae2ed5ac0b4ba2bb534871dd960e99ae1638ca7da6bab934a5dd022R122). Could we do something like that here?

```suggestion

*default: `true`*

```